### PR TITLE
[SDK][MC] errcodes.mc: Added more COM error messages

### PR DESCRIPTION
**Pull request by whindsaks**

### CODE REVIEW

File: sdk/include/reactos/mc/errcodes.mc [hunk 1]
```diff
@@ -43001,7 +43121,7 @@ Severity=Warning
 Facility=WIN32
 SymbolicName=E_ACCESSDENIED
 Language=English
-WIN32 access denied error
+General access denied error
```
Comment (SergeGautherie) [inline sdk/include/reactos/mc/errcodes.mc:43124]: Update all languages.

**Comment by binarymaster** [comment]
Fixed module prefix in PR title, please take into account on merge.

File: sdk/include/reactos/mc/errcodes.mc (hunk 1 above)
Comment (whindsaks) [reply sdk/include/reactos/mc/errcodes.mc:43124]: I don't know what the correct translation on these systems is.

Comment (binarymaster) [reply sdk/include/reactos/mc/errcodes.mc:43124]: You can revert to English in such cases.

File: sdk/include/reactos/mc/errcodes.mc [hunk 2]
```diff
@@ -43001,16 +43121,16 @@ Severity=Warning
...
 Language=Polish
-WIN32 access denied error
+General access denied error
 .
 Language=Romanian
-WIN32 access denied error
+General access denied error
 .
 Language=Japanese
 WIN32のアクセス拒否エラーです。
```
Comment (SergeGautherie) [inline sdk/include/reactos/mc/errcodes.mc:43136]: Update this one too.

Comment (katahiromz) [reply sdk/include/reactos/mc/errcodes.mc:43136]:
```suggestion
一般的なアクセス拒否エラーです。
```

**Review by binarymaster — APPROVED** [review]

**Review by HBelusca — APPROVED** [review]

**Pull request merged**